Gemfile

```diff
@@ -43,7 +43,7 @@ gem "jquery-ui-rails"
 gem "simple_form", '~> 3.1.0'
 gem 'nokogiri'
 gem 'equivalent-xml'
-gem 'solr_wrapper'
+gem 'solr_wrapper', '0.7.3'
```

Comment:
If it's anything like what we saw in other hydra projects, it's simply fixing units => distanceUnits (deprecated in Solr 5.x, removed in 6.x):

https://github.com/projecthydra/active_fedora/commit/f492fafbca20aa1e69546858437fb8f07277f158